config/stacks/shared.ts

```diff
@@ -94,9 +94,12 @@ const getEventSourceConstructProps = (stage: AppStage): EventSourceProps => {
     rules: [
       {
         bucket: oncoanalyserBucket[stage],
+        eventTypes: ['Object Created', 'Object Deleted'],
```

Comment:
Are those the only event types the FileManager is currently monitoring? 
If yes, that's OK, just be aware that this is a while-listing and you'll miss any other events (which is OK for the portal use case, but you'll have to judge if it's also OK for the FileManager).

Reply:
Yep, those are the only ones considered right now. It derives all the other information from `HeadObject` and `GetObjectTagging`. In the future, I can add support for "Object Storage Class Changed" and "Object Tags Added" to avoid using an API call.

Reply:
Sound good. Go for it, Marko. Pls merge when you ready.

Reply:
Adding to this issue: https://github.com/umccr/orcabus/issues/322#issuecomment-2443343299